pkg/web/apps/apps_psql_repository.go

```diff
@@ -36,3 +38,55 @@ func (a *appsPostgreSQLRepository) GetApps(c echo.Context) (*[]models.App, error
 
 	return &apps, nil
 }
+
```

Comment:
What is the idea of apps_psql_repository? Is it to connect in the database or is it the apps model definition?
If it is to connect to the database then I think we should do as describe [here](https://astaxie.gitbooks.io/build-web-application-with-golang/en/05.4.html).

If it is the model class/gorountine to get the data then

Reply:
see https://github.com/aerogear/mobile-security-service/pull/17 for reasons